class.jetpack.php

```diff
@@ -447,6 +448,8 @@ private function __construct() {
 		$this->sync->mock_option( 'wp_version', array( 'Jetpack', 'get_wp_version' ) );
 
 		add_action( 'init', array( $this, 'sync_update_data') );
+		add_action( 'init', array( $this, 'sync_theme_data' ) );
```

Comment:
I think it might be best to just move the contents of sync_theme_data function to the constructor. 
Since we have a whole bunch of the $this->sync->mock_option() there already.